### PR TITLE
docs(configuration options): rewrite `keepUpdatedLabel`

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes

- Rewrite `keepUpdatedLabel` docs
- Rewrite description field for `keepUpdatedLabel` in code

## Context

Follow up after:

- #27542

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/configuration-options.md [hunk 1]
```diff
@@ -2125,12 +2125,13 @@ We recommend that you use the `strict` mode, and enable the `dependencyDashboard
...
+On supported platforms you may add a label to a PR so that Renovate recreates/rebases the PR when the branch falls behind the base branch.
 Adding the `keepUpdatedLabel` label to a PR makes Renovate behave as if `rebaseWhen` were set to `behind-base-branch`, but only for the given PR.
-The label is not removed from the PR after the rebase is complete, unlike what happens with `rebaseLabel`.
+Renovate does _not_ remove the label from the PR after it finishes rebasing.
+This is different from the `rebaseLabel` option, where Renovate _removes_ the label from the PR after rebasing.
 
-This can be useful when you have approved certain PRs and want to keep them updated until they are ready to be merged.
-The setting `keepUpdatedLabel` is best used in conjunction with `rebaseWhen` set to the values of `never` or `conflicted` that limit rebasing.
+`keepUpdatedLabel` can be useful when you have approved certain PRs and want Renovate to keep the PRs up-to-date until you're ready to merge them.
+The setting `keepUpdatedLabel` is best used together with `rebaseWhen` set to the values of `never` or `conflicted` that limit rebasing. ???
```
Comment (HonkingGoose) [inline docs/usage/configuration-options.md:2134]: This sentence is hard to understand for me. I think we're missing some connecting sentence that explains why you should use `rebaseWhen=never/conflicted`when you use `keepUpdatedLabel`.

Be sure to remove the `???` I added. I added `???` so I can comment on this line directly. 😄

File: docs/usage/configuration-options.md [hunk 2]
```diff
@@ -2125,12 +2125,13 @@ We recommend that you use the `strict` mode, and enable the `dependencyDashboard
 
 ## keepUpdatedLabel
```
Comment (HonkingGoose) [inline docs/usage/configuration-options.md:2126]: On a related note, the `description` for this config option is hard to understand also:

> Label to request Renovate bot always rebase to keep branch updated.

Comment (rarkins) [reply docs/usage/configuration-options.md:2126]: "If set, users can add this label to PRs to request they be kept updated with the base branch." ?

File: docs/usage/configuration-options.md (hunk 1 above)
Comment (rarkins) [inline docs/usage/configuration-options.md:2134]:
```suggestion
The setting `keepUpdatedLabel` is best used in this scenario:

- By default, you configure `rebaseWhen` to `never` or `conflicted` to reduce rebasing
- Sometimes, you want Renovate to keep specific PRs up-to-date with their base branch (equivalent to `rebaseWhen=behind-base-branch`)
```

**Review by rarkins — CHANGES_REQUESTED** [review]

File: docs/usage/configuration-options.md (hunk 2 above)
Comment (rarkins) [reply docs/usage/configuration-options.md:2126]: Do you want to make the change in this PR?

File: lib/config/options/index.ts [hunk 3]
```diff
@@ -1766,7 +1766,7 @@ const options: RenovateOptions[] = [
   {
     name: 'keepUpdatedLabel',
     description:
-      'Label to request Renovate bot always rebase to keep branch updated.',
+      'If set, users can add this label to PRs to request they be kept updated with the base branch.',
```
Comment (HonkingGoose) [inline lib/config/options/index.ts:1769]: Does this change make this a `fix` type PR? Or can I keep using the `docs` scope for it?

Comment (rarkins) [reply lib/config/options/index.ts:1769]: docs scope

**Review by rarkins — APPROVED** [review]

**Pull request merged**